src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs

```diff
@@ -401,7 +407,7 @@ private async ValueTask<IInstalledAndTransitivePackages> GetInstalledAndTransiti
 
             if (itemFilter == ItemFilter.Installed)
             {
-                if (await ExperimentUtility.IsTransitiveOriginExpEnabled.GetValueAsync(cancellationToken))
+                if (await ExperimentUtility.IsTransitiveOriginExpEnabled.GetValueAsync(cancellationToken) && !isSolution)
```

Comment:
nit: Normally, I'd put the isSolution check first because it's not a compute one.